hs_model_program/models.py

```diff
@@ -169,7 +173,7 @@ def get_xml(self, pretty_print=True, include_format_elements=True):
 
 
         # get the xml string for Model Program
-        xml_string = super(ModelProgramMetaData, self).get_xml(pretty_print=pretty_print)
+        xml_string = super(ModelProgramMetaData, self).get_xml_legacy(pretty_print=pretty_print)
```

Comment:
Why legacy? Because there's another PR in planning? Some other reason?

Reply:
legacy is for resource types outside of Composite/Collections.  I did not think it was worth the effort to support the other resource types as they should be replaced by aggregations eventually.
https://github.com/hydroshare/hydroshare/pull/3928#discussion_r530439375

Reply:
OK. This is fine.